### PR TITLE
Add Resident Evil 4 to ASLs (by LetsGetPitted)

### PR DESCRIPTION
**Pull request by drtchops**
Pitted did all the work, I'm just hosting the file for him.

### CODE REVIEW

**Comment by wooferzfg1** [comment]
Apparently one of these games already has an Auto Splitter?

**Comment by drtchops** [comment]
5 and REmake do, this is for 4.

**Comment by wooferzfg1** [comment]
Oh, nvm, the problem is that you can't add both RE4 and re4. LiveSplit isn't case sensitive with the Auto Splitter game names in 1.6.

**Comment by drtchops** [comment]
Oh, good.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -374,6 +374,19 @@
   </AutoSplitter>
   <AutoSplitter>
     <Games>
+      <Game>Resident Evil 4</Game>
+      <Game>Biohazard 4</Game>
+      <Game>RE4</Game>
+      <Game>re4</Game>
```
Comment (wooferzfg1) [inline LiveSplit.AutoSplitters.xml]: This needs to be fixed though. It should have either RE4 or re4, not both.

**Pull request merged**